clients/clojure/github.com/ptaoussanis/carmine.json

```diff
@@ -1,6 +1,6 @@
 {
     "name": "carmine",
-    "description": "Simple, high-performance Redis (2.0+) client for Clojure.",
+    "description": "Simple, high-performance Valkey (2.0+) client for Clojure.",
```

Comment:
There is no Valkey 2.0. It probably works also for Redis so that's what they mean here. I'm not sure, maybe we can say it's a client for Valkey and Redis 2.0+.